WeakAurasOptions/AuthorOptions.lua

```diff
@@ -975,6 +975,35 @@ typeControlAdders = {
       step = 1
     }
   end,
+  media = function(options, args, data, order, prefix, i)
+    local option = options[i]
+    args[prefix .. "mediaType"] = {
+      type = "select",
+      width = WeakAuras.normalWidth,
+      name = name(option, "mediaType", L["Media Type"]),
+      desc = desc(option, "mediaType"),
+      values = OptionsPrivate.Private.shared_media_types,
+      order = order(),
+      get = get(option, "mediaType"),
+      set = set(data, option, "mediaType")
+    }
+    args[prefix .. "default"] = {
+      type = "select",
+      width = WeakAuras.normalWidth,
+      name = name(option, "default", L["Default"]),
+      desc = desc(option, "default"),
+      values = function()
+        if option.mediaType == "sound" then
+          return OptionsPrivate.Private.sound_types
+        else
+          return AceGUIWidgetLSMlists[option.mediaType]
+        end
+      end,
+      order = order(),
```

Comment:
suggestion for using LSM widgets
start of the "media" function add
```Lua
    local controls = {
      statusbar = "LSM30_Statusbar",
      border = "LSM30_Border",
      background = "LSM30_Background",
      font = "LSM30_Font"
    }
```
and add this in args[prefix.."default"]
```Lua
dialogControl = controls[option.mediaType],
```